core/editor/editor.php

```diff
@@ -589,8 +589,11 @@ public function enqueue_scripts() {
 				'darkModeStylesheetURL' => ELEMENTOR_ASSETS_URL . 'css/editor-dark-mode' . $suffix . '.css',
 				'defaultGenericFonts' => $kits_manager->get_current_settings( 'default_generic_fonts' ),
 			],
-			// Empty array for BC to avoid errors.
-			'i18n' => [],
+			// Array for BC to avoid errors.
+			'i18n' => [
```

Comment:
Instead of that, put this string in the pro `i18n` config and use `elementorPro.translate( 'edit_element', [ documentTitle ] ) )`